src/posthog-core.js

```diff
@@ -640,7 +640,7 @@ PostHogLib.prototype._invokeCaptureHooks = function (eventName, eventData) {
 
 PostHogLib.prototype._calculate_event_properties = function (event_name, event_properties, start_timestamp) {
     // set defaults
-    let properties = event_properties || {}
+    let properties = { ...event_properties } || {}
```

Comment:
Well, I guess in this case `|| {}` does nothing.

Reply:
True, removed